src/components/Input/RadioButton/RadioButton.js

```diff
@@ -34,12 +34,15 @@ const RadioWrapper = styled.label`
   }
 `;
 
-const RadioText = styled.span`
+const RadioText = styled.div`
   display: flex;
   outline: none;
-  align-items: center;
-  justify-content: center;
   margin-left: 7px;
+  ${props =>
+    props.isCombined
```

Comment:
I am just wondering if we can improve the naming of this prop. Something that is more related to the radio button layout. How does it sound `isTopAligned`?

Reply:
Renamed.